### PR TITLE
Awesome books: refactor to use JavaScript classes

### PR DESCRIPTION
**Pull request by IjayAbby**
In this project I was to complete the following instructions.
- Use JavaScript classes instead of objects and arrays.
- You can use objects and arrays, but do it inside classes.
- Create class methods to add and remove books.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,116 +1,135 @@
...
-}
+  static removeBook(index) {
+    const books = Store.getBooks();
 
-/* eslint-disable no-use-before-define */
-function saveBooks() {
-  setItem('books', JSON.stringify(books));
-}
+    // books.forEach((book, index) => {
+    //   if (index === i) {
```
Comment (Davidosky007) [inline app.js]: * Kindly remove commented code from your codebase it makes your code bulky and distracts one from the actual code [Read more here](https://kentcdodds.com/blog/please-dont-commit-commented-out-code)

File: index.html [hunk 2]
```diff
@@ -1,25 +1,39 @@
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
-   <meta charset="utf-8">
-   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-   <title>Awesome Books</title>
-
-   <link rel="stylesheet" href="index.css">
+    <meta charset="UTF-8">
```
Comment (Davidosky007) [inline index.html:4]: * Your Readme file is missing please fix it 🙏 as it is important you have a readme so people know more about the project

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @IjayAbby, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
cr2 gif_almost-there

## Required Changes ♻️
## _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by hemantso — APPROVED** [review]
Hi @IjayAbby, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 

Alt Text

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Kindly try to create the object using the instance of class rather than directly calling the static method on the class directly.

Cheers,
Hemant Soni
Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**